hack/preload-images/generate-preloaded-images-tar.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
how hard would to convert this to a go script ?

Reply:
done